### ISSUE TITLE
(#210) Build & theme Annual meetings landing page

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/alnap#35]
### Dependencies
- [x]  Scoping annual meetings page needs to be done first  #65 / #30
---
### Done criteria
- [x] Expand/collapse more text nugget
- [ ] Show all sorted in reverse chronological order with load more
- [x] Promo
---
### Designs
- [Events wide](https://invis.io/VPC8MGQ6E)
- [Annual meetings landing](https://invis.io/T2C3JAS79)
---
### ALNAP to check
http://alnap-dev.mrmdev.co.uk/upcoming-events/annual-meetings
- [x] Looks like wireframe/design

**Comment by angievanhegan** [comment]
@melkers Sorry Dave, you're going to have to spell out for me what we needed to clarify with the client. Thanks.

**Comment by melkers** [comment]
@angievanhegan here's the annual meeting landing page which basically provides a link to to each previous annual meeting.  What we need to decide is how we deal with annual meetings, for which I'm going to create another card.

**Comment by angievanhegan** [comment]
@dave-manta Do we need to know what goes on the annual meeting page before we can build this? Also, if it's a landing page, should it be in this sprint or can we pop it back into the backlog for now?

**Comment by dave-manta** [comment]
@angievanhegan yes, I think we do need to decide on how we approach the annual meetings.  I would put it in the backlog.

**Comment by Maria-alnap** [comment]
@nadia847 this looks good. I'll move to done!
